src/Tribe/RSVP.php

```diff
@@ -430,6 +579,178 @@ public function get_statuses_by_action( $action ) {
 		return $status_mgr->get_statuses_by_action( $action, 'rsvp' );
 	}
 
+	/**
+	 * Create an attendee for a RSVP ticket.
+	 *
+	 * @since TBD
+	 *
+	 * @param \Tribe__Tickets__Ticket_Object $ticket        Ticket object.
+	 * @param array                          $attendee_data Attendee data.
+	 *
+	 * @return int Attendee ID.
+	 *
+	 * @throws Exception
```

Comment:
:no_entry_sign: Comment missing for @throws tag in function comment

<sub>hash: 59b00805ebfd2d79523127cd2c2d5ef7</sub>